app/graphql/mutations/base_mutation.rb

```diff
@@ -46,6 +46,21 @@ def find_profile(profile_id, permission: :edit?)
 
   # Common class for all GraphQL mutations
   class BaseMutation < GraphQL::Schema::RelayClassicMutation
+    class << self
+      def authorized?(object, context)
+        permission = object.class.instance_variable_get(:@rbac_permission)
+        if permission.nil?
+          raise GraphQL::UnauthorizedError
+        end
+
+        context[:current_user].authorized_to?(permission)
+      end
+
+      def enforce_rbac(*permission)
```

Comment:
According to this, you're making an array, but the check above deals with a single item.